[Automation] javac compile fails for com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0

## javac compile on ubuntu-latest / JDK 25 / mode current-defaults

Failure kind: javac compile
OS: ubuntu-latest
JDK: 25
Native-image mode: current-defaults
Reproducer: `env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/26742420600
Last 300 lines of the log:
```console
========================================================================================
 run-consecutive-tests input
========================================================================================
Coordinates: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
Raw versions JSON: [
  "2.22.0"
]
Parsed version count: 1
Parsed versions: 2.22.0
Timeout per Gradle invocation: 5m
========================================================================================
 Testing com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5:2.22.0
========================================================================================
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5:2.22.0 stage native-image run
========================================================================================
Starting stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:16Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
Starting a Gradle Daemon, 1 busy Daemon could not be reused, use --status for details
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 7s
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :tckTest
====================
Testing library: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':nativeTestCompile'.
> Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
   > Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or 
Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
8 actionable tasks: 8 up-to-date
--debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 3s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 11s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:28Z with exit code 1 after 12s
❌ ERROR: 'native-image run' for 2.22.0 failed with exit code 1.
SOFT-FAIL[native-image run][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5]
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5:2.22.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:28Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 931ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':compileTestJava'.
> Could not resolve all dependencies for configuration ':testCompileClasspath'.
   > Could not resolve all dependencies for configuration ':testCompileClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Searched in the following locations:
          - file:/home/runner/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
          - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 1s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
8 actionable tasks: 8 up-to-date

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:31Z with exit code 1 after 3s
❌ ERROR: 'javac compile' for 2.22.0 failed with exit code 1.
FAILED[javac compile][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5]
```

## javac compile on ubuntu-latest / JDK latest-ea / mode current-defaults

Failure kind: javac compile
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: current-defaults
Reproducer: `env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/26742420600
Last 300 lines of the log:
```console
========================================================================================
 run-consecutive-tests input
========================================================================================
Coordinates: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
Raw versions JSON: [
  "2.22.0"
]
Parsed version count: 1
Parsed versions: 2.22.0
Timeout per Gradle invocation: 5m
========================================================================================
 Testing com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5:2.22.0
========================================================================================
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5:2.22.0 stage native-image run
========================================================================================
Starting stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:16Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
Starting a Gradle Daemon, 1 busy Daemon could not be reused, use --status for details
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 7s
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :tckTest
====================
Testing library: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':nativeTestCompile'.
> Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
   > Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 2s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
8 actionable tasks: 8 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 11s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:27Z with exit code 1 after 11s
❌ ERROR: 'native-image run' for 2.22.0 failed with exit code 1.
SOFT-FAIL[native-image run][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5]
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5:2.22.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:27Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 893ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':compileTestJava'.
> Could not resolve all dependencies for configuration ':testCompileClasspath'.
   > Could not resolve all dependencies for configuration ':testCompileClasspath'.

      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Searched in the following locations:
          - file:/home/runner/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
          - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
8 actionable tasks: 8 up-to-date
* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 926ms

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:30Z with exit code 1 after 3s
❌ ERROR: 'javac compile' for 2.22.0 failed with exit code 1.
FAILED[javac compile][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5]
```

## javac compile on ubuntu-latest / JDK latest-ea / mode future-defaults-all

Failure kind: javac compile
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: future-defaults-all
Reproducer: `env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=future-defaults-all ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/26742420600
Last 300 lines of the log:
```console
========================================================================================
 run-consecutive-tests input
========================================================================================
Coordinates: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
Raw versions JSON: [
  "2.22.0"
]
Parsed version count: 1
Parsed versions: 2.22.0
Timeout per Gradle invocation: 5m
========================================================================================
 Testing com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5:2.22.0
========================================================================================
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5:2.22.0 stage native-image run
========================================================================================
Starting stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:19Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=future-defaults-all ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
Starting a Gradle Daemon, 1 busy Daemon could not be reused, use --status for details
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 7s
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :tckTest
====================
Testing library: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':nativeTestCompile'.
> Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
   > Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.

        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 3s
Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
8 actionable tasks: 8 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 12s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:31Z with exit code 1 after 12s
❌ ERROR: 'native-image run' for 2.22.0 failed with exit code 1.
SOFT-FAIL[native-image run][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=future-defaults-all ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5]
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5:2.22.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:32Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=future-defaults-all ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 959ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.18.5/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':compileTestJava'.
> Could not resolve all dependencies for configuration ':testCompileClasspath'.
   > Could not resolve all dependencies for configuration ':testCompileClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Searched in the following locations:
          - file:/home/runner/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
          - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 995ms

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
8 actionable tasks: 8 up-to-date

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:35Z with exit code 1 after 3s
❌ ERROR: 'javac compile' for 2.22.0 failed with exit code 1.
FAILED[javac compile][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=future-defaults-all ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5]
```

--- a/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/reachability-metadata.json
+++ b/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/reachability-metadata.json
@@ -1,0 +1,34 @@
+{
+  "reflection": [
+    {
+      "condition": {
+        "typeReached": "com.fasterxml.jackson.jakarta.rs.json.JsonMapperConfigurator"
+      },
+      "type": "com.fasterxml.jackson.databind.ext.Java7SupportImpl"
+    },
+    {
+      "condition": {
+        "typeReached": "com.fasterxml.jackson.jakarta.rs.json.JsonMapperConfigurator"
+      },
+      "type": "com.fasterxml.jackson.module.jakarta.xmlbind.JakartaXmlBindAnnotationIntrospector",
+      "methods": [
+        {
+          "name": "<init>",
+          "parameterTypes": []
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "com.fasterxml.jackson.jakarta.rs.json.JsonMapperConfigurator"
+      },
+      "type": "java.lang.String[]"
+    },
+    {
+      "condition": {
+        "typeReached": "com.fasterxml.jackson.jakarta.rs.json.JsonMapperConfigurator"
+      },
+      "type": "sun.util.resources.cldr.CalendarData"
+    }
+  ]
+}

--- a/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/index.json
+++ b/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/index.json
@@ -1,6 +1,15 @@
 [
   {
     "latest" : true,
+    "metadata-version" : "2.22.0",
+    "tested-versions" : [
+      "2.22.0"
+    ],
+    "allowed-packages" : [
+      "com.fasterxml.jackson.jakarta.rs.json"
+    ]
+  },
+  {
     "metadata-version" : "2.18.5",
     "source-code-url" : "https://repo1.maven.org/maven2/com/fasterxml/jackson/jakarta/rs/jackson-jakarta-rs-json-provider/$version$/jackson-jakarta-rs-json-provider-$version$-sources.jar",
     "repository-url" : "https://github.com/FasterXML/jackson-jakarta-rs-providers",

--- a/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/index.json
+++ b/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/index.json
@@ -2,6 +2,11 @@
   {
     "latest" : true,
     "metadata-version" : "2.22.0",
+    "source-code-url" : "https://repo1.maven.org/maven2/com/fasterxml/jackson/jakarta/rs/jackson-jakarta-rs-json-provider/$version$/jackson-jakarta-rs-json-provider-$version$-sources.jar",
+    "repository-url" : "https://github.com/FasterXML/jackson-jakarta-rs-providers",
+    "test-code-url" : "https://github.com/FasterXML/jackson-jakarta-rs-providers/tree/jackson-jakarta-rs-providers-$version$/json/src/test",
+    "documentation-url" : "https://repo1.maven.org/maven2/com/fasterxml/jackson/jakarta/rs/jackson-jakarta-rs-json-provider/$version$/jackson-jakarta-rs-json-provider-$version$-javadoc.jar",
+    "description" : "Jackson Jakarta-RS JSON Provider integrates Jackson data binding with Jakarta RESTful Web Services implementations to read and write JSON request and response bodies. It supplies JSON MessageBodyReader and MessageBodyWriter providers, automatic provider registration, and Jackson-specific configuration support for REST endpoints.",
     "tested-versions" : [
       "2.22.0"
     ],

--- a/stats/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/execution-metrics.json
+++ b/stats/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/execution-metrics.json
@@ -1,0 +1,99 @@
+{
+  "fix_javac_fail:2026-06-05": {
+    "timestamp": "2026-06-05T19:06:32.336807Z",
+    "library": "com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0",
+    "starting_commit": "9d15d0854cecfec7b970f60554fa95734f175ab7",
+    "ending_commit": "9934aa693326e15bd929476bc5bfaed5bef93a4e",
+    "previous_library": "com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.18.5",
+    "strategy_name": "javac_iterative_with_coverage_sources_pi_gpt-5.5",
+    "agent": "pi",
+    "model": "oca/gpt-5.5",
+    "status": "success",
+    "stats": {
+      "version": "2.22.0",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 2,
+            "totalCalls": 2
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 2,
+        "totalCalls": 2
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 109,
+          "missed": 333,
+          "ratio": 0.246606,
+          "total": 442
+        },
+        "line": {
+          "covered": 24,
+          "missed": 84,
+          "ratio": 0.222222,
+          "total": 108
+        },
+        "method": {
+          "covered": 4,
+          "missed": 29,
+          "ratio": 0.121212,
+          "total": 33
+        }
+      }
+    },
+    "previous_library_stats": {
+      "version": "2.18.5",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 2,
+            "totalCalls": 2
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 2,
+        "totalCalls": 2
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 105,
+          "missed": 343,
+          "ratio": 0.234375,
+          "total": 448
+        },
+        "line": {
+          "covered": 23,
+          "missed": 88,
+          "ratio": 0.207207,
+          "total": 111
+        },
+        "method": {
+          "covered": 4,
+          "missed": 29,
+          "ratio": 0.121212,
+          "total": 33
+        }
+      }
+    },
+    "metrics": {
+      "input_tokens_used": 94513,
+      "cached_input_tokens_used": 734720,
+      "output_tokens_used": 8959,
+      "iterations": 2,
+      "cost_usd": 0.6362,
+      "tested_library_loc": 24,
+      "metadata_entries": 5,
+      "previous_library_metadata_entries": 10,
+      "code_coverage_percent": 22.22,
+      "previous_library_coverage_percent": 20.72
+    },
+    "artifacts": {
+      "test_file": "tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/src/test/java/com_fasterxml_jackson_jakarta_rs/jackson_jakarta_rs_json_provider/JsonMapperConfiguratorTest.java",
+      "metadata_file": "metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/reachability-metadata.json"
+    }
+  }
+}

--- a/stats/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/stats.json
+++ b/stats/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/stats.json
@@ -1,0 +1,37 @@
+{
+  "versions" : [ {
+    "version" : "2.22.0",
+    "dynamicAccess" : {
+      "breakdown" : {
+        "reflection" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 2,
+          "totalCalls" : 2
+        }
+      },
+      "coverageRatio" : 1.0,
+      "coveredCalls" : 2,
+      "totalCalls" : 2
+    },
+    "libraryCoverage" : {
+      "instruction" : {
+        "covered" : 109,
+        "missed" : 333,
+        "ratio" : 0.246606,
+        "total" : 442
+      },
+      "line" : {
+        "covered" : 24,
+        "missed" : 84,
+        "ratio" : 0.222222,
+        "total" : 108
+      },
+      "method" : {
+        "covered" : 4,
+        "missed" : 29,
+        "ratio" : 0.121212,
+        "total" : 33
+      }
+    }
+  } ]
+}

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/.gitignore
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/.gitignore
@@ -1,0 +1,4 @@
+gradlew.bat
+gradlew
+gradle/
+build/

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/build.gradle
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/build.gradle
@@ -1,0 +1,27 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = tck.testedLibraryVersion.get()
+
+dependencies {
+    testImplementation "com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:$libraryVersion"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+}
+
+graalvmNative {
+    agent {
+        defaultMode = "conditional"
+        modes {
+            conditional {
+                userCodeFilterPath = "user-code-filter.json"
+            }
+        }
+    }
+}

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/gradle.properties
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/gradle.properties
@@ -1,0 +1,6 @@
+# Optional explicit tested library coordinates (format: group:artifact:version).
+# If omitted, fallback resolution in org.graalvm.internal.tck.gradle still uses
+# environment variable GVM_TCK_LC and then this property.
+library.coordinates = com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:2.22.0
+library.version = 2.22.0
+metadata.dir = com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/settings.gradle
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/settings.gradle
@@ -1,0 +1,20 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+pluginManagement {
+    def tckPath = Objects.requireNonNullElse(
+            System.getenv("GVM_TCK_TCKDIR"),
+            "../../../../tck-build-logic"
+    )
+    includeBuild(tckPath)
+    repositories { mavenLocal(); gradlePluginPortal() }
+}
+
+plugins {
+    id "org.graalvm.internal.tck-settings" version "1.0.0-SNAPSHOT"
+}
+
+rootProject.name = 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-json-provider_tests'

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/src/test/java/com_fasterxml_jackson_jakarta_rs/jackson_jakarta_rs_json_provider/JsonMapperConfiguratorTest.java
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/src/test/java/com_fasterxml_jackson_jakarta_rs/jackson_jakarta_rs_json_provider/JsonMapperConfiguratorTest.java
@@ -1,0 +1,42 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_json_provider;
+
+import com.fasterxml.jackson.databind.AnnotationIntrospector;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.jakarta.rs.cfg.Annotations;
+import com.fasterxml.jackson.jakarta.rs.json.JsonMapperConfigurator;
+import com.fasterxml.jackson.module.jakarta.xmlbind.JakartaXmlBindAnnotationIntrospector;
+import jakarta.xml.bind.annotation.XmlElement;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JsonMapperConfiguratorTest {
+    @Test
+    void configuresDefaultMapperWithJakartaXmlBindAnnotations() throws Exception {
+        JsonMapperConfigurator configurator = new JsonMapperConfigurator(null,
+                new Annotations[] {Annotations.JAKARTA_XML_BIND});
+
+        ObjectMapper mapper = configurator.getDefaultMapper();
+
+        assertThat(mapper.getSerializationConfig().getAnnotationIntrospector().allIntrospectors())
+                .extracting(AnnotationIntrospector::getClass)
+                .contains(JakartaXmlBindAnnotationIntrospector.class);
+        assertThat(mapper.writeValueAsString(new XmlElementBean("configured")))
+                .isEqualTo("{\"xml_name\":\"configured\"}");
+    }
+
+    public static final class XmlElementBean {
+        @XmlElement(name = "xml_name")
+        public final String name;
+
+        public XmlElementBean(String name) {
+            this.name = name;
+        }
+    }
+}

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/src/test/java/com_fasterxml_jackson_jakarta_rs/jackson_jakarta_rs_json_provider/JsonMapperConfiguratorTest.java
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/src/test/java/com_fasterxml_jackson_jakarta_rs/jackson_jakarta_rs_json_provider/JsonMapperConfiguratorTest.java
@@ -6,8 +6,10 @@
  */
 package com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_json_provider;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.AnnotationIntrospector;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector;
 import com.fasterxml.jackson.jakarta.rs.cfg.Annotations;
 import com.fasterxml.jackson.jakarta.rs.json.JsonMapperConfigurator;
 import com.fasterxml.jackson.module.jakarta.xmlbind.JakartaXmlBindAnnotationIntrospector;
@@ -17,6 +19,20 @@ import org.junit.jupiter.api.Test;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class JsonMapperConfiguratorTest {
+    @Test
+    void configuresDefaultMapperWithJacksonAnnotations() throws Exception {
+        JsonMapperConfigurator configurator = new JsonMapperConfigurator(null,
+                new Annotations[] {Annotations.JACKSON});
+
+        ObjectMapper mapper = configurator.getDefaultMapper();
+
+        assertThat(mapper.getSerializationConfig().getAnnotationIntrospector().allIntrospectors())
+                .extracting(AnnotationIntrospector::getClass)
+                .contains(JacksonAnnotationIntrospector.class);
+        assertThat(mapper.writeValueAsString(new JsonPropertyBean("configured")))
+                .isEqualTo("{\"json_name\":\"configured\"}");
+    }
+
     @Test
     void configuresDefaultMapperWithJakartaXmlBindAnnotations() throws Exception {
         JsonMapperConfigurator configurator = new JsonMapperConfigurator(null,
@@ -29,6 +45,15 @@ public class JsonMapperConfiguratorTest {
                 .contains(JakartaXmlBindAnnotationIntrospector.class);
         assertThat(mapper.writeValueAsString(new XmlElementBean("configured")))
                 .isEqualTo("{\"xml_name\":\"configured\"}");
+    }
+
+    public static final class JsonPropertyBean {
+        @JsonProperty("json_name")
+        public final String name;
+
+        public JsonPropertyBean(String name) {
+            this.name = name;
+        }
     }
 
     public static final class XmlElementBean {

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/user-code-filter.json
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/2.22.0/user-code-filter.json
@@ -1,0 +1,13 @@
+{
+  "rules" : [
+    {
+      "excludeClasses" : "**"
+    },
+    {
+      "includeClasses" : "com.fasterxml.jackson.jakarta.rs.json.**"
+    },
+    {
+      "includeClasses" : "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_json_provider.**"
+    }
+  ]
+}

